src/test/resources/META-INF/mods.toml

```diff
@@ -148,5 +150,3 @@ license="LGPL v2.1"
     modId="tag_based_tool_types"
 [[mods]]
     modId="check_spawn_event_test"
-
-# ADD ABOVE THIS LINE
```

Comment:
This should not be being removed...

Reply:
I added that to avoid having every single PR swap this between "line break at end of file" and "no line break at end of file".
It's technically not necessary, just removes that annoyance.